src/components/MapFacetContents.js

```diff
@@ -200,14 +200,29 @@ class MapFacetContents extends React.Component<MapFacetContentsDefaultProps, Map
 
       const points = this.props.buckets.map((bucket) => {
         const value = bucket.value; // JSON.parse(bucket.value);
```

Comment:
We should probably delete this old comment as long as we're changing the file...